content/20-exploring-the-cluster/21-exploring-the-cluster/index.en.md

```diff
@@ -26,93 +25,96 @@ At the top of the page, select the search bar and search for **`"CloudFormation"
 
 ### Step 2:
 
-In the **`CloudFormation Console`**, click on the Rancher Manager **`Stack`**. If you are using AWS Workshop Studio at an AWS Event, the stack will be named **`rke2-eks-cluster-workshop`**. Outside AWS Workshop Studio, the stack will have a user-defined name from when it was created in the previous section.
+In the **`CloudFormation Console`**, click on the Rancher Manager **`Stack`**. If you are using AWS Workshop Studio at an AWS Event, the stack will be named **`rke2-eks-cluster-workshop`**. Outside AWS Workshop Studio, the stack will have a user-defined name you selected in the previous section.
+
+On the stack detail page, click on the **`Outputs`** tab. You should see the **`RancherURL`**. Open the link in a new tab to bring up the Rancher Multi-Cluster Manager web interface. If your browser presents you with a certificate warning, you can safely proceed.
 
-Inside of the stack, click on the **`Outputs`** tab. You should see the **`RancherURL`**, click the link in a new tab to bring up the Rancher Multi-Cluster Manager. If you a presented with a certificate warning, you can safely proceed since the workshop environment uses self-signed certificates for learning purposes. In a production environment, always use certificates issued from a trusted Certificate Authority. [AWS Certificate Manager (ACM)](https://aws.amazon.com/certificate-manager/) makes it easy to provision, manage, deploy, and renew SSL/TLS certificates.
+> The workshop environment uses self-signed certificates for learning purposes. In a production environment, always use certificates issued from a trusted Certificate Authority. [AWS Certificate Manager (ACM)](https://aws.amazon.com/certificate-manager/) makes it easy to provision, manage, deploy, and renew SSL/TLS certificates.
```

Comment:
Change to "The workshop environment uses self-signed certificates for learning purposes. In a production environment, always use certificates issued from a trusted public or private certificate authority. [AWS Certificate Manager (ACM)](https://aws.amazon.com/certificate-manager/) makes it easy to provision, manage, deploy, and renew SSL/TLS certificates.